q2_assembly/quast/tests/test_quast.py

```diff
@@ -12,13 +12,18 @@
 import shutil
 import tempfile
 import unittest
+import warnings
 from filecmp import dircmp
 from subprocess import CalledProcessError
 from unittest.mock import ANY, MagicMock, Mock, call, mock_open, patch
 from zipfile import ZipFile
 
 import pandas as pd
+from future.moves import itertools
+from globus_sdk.experimental.auth_requirements_error import ValidationError
```

Comment:
And why on earth would we want to use this validation error? There is a QIIME-built-in one you should be using (please pay attention to PyCharm autosuggested imports).